dev-libs/tllist/tllist-1.0.5.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Header-only implementation of a typed linked list in C"
+HOMEPAGE="https://codeberg.org/dnkl/tllist"
+SRC_URI="https://codeberg.org/dnkl/tllist/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+src_install() {
+	local DOCS=( README.md )
```

Comment:
README* is installed by default if DOCS is unset (only needed when there's other files like CHANGELOG.md), so you don't need to set it here (just keep deleting the directory)